File: code/1038/1038.cpp
#include <iomanip>
#include <iostream>
using namespace std;

int main() {

  double prices[] = {4, 4.5, 5, 2, 1.5};

  int productCode, quantity;

  cin >> productCode >> quantity;

  cout << "Total: R$ " << fixed << setprecision(2)
       << prices[productCode - 1] * quantity << endl;

  return 0;
}

File: code/1038/1038.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const prices = [4, 4.5, 5, 2, 1.5];

const productCode = parseInt(lines[0]);
const quantity = parseInt(lines[1]);

console.log(`Total: R\$ ${(prices[productCode - 1] * quantity).toFixed(2)}`);
